### ISSUE TITLE
Source member sometimes truncates text after column 80

### ISSUE THREAD

**Issue by richardm90** [codefori/vscode-ibmi#1235]
Follow on from issue #1214 

Code for IBM i v1.7.12. I have source dates enabled with a tracking mode of diff.

I connect to my IBM i server in VS Code.

I then create a new source member in QRPGLESRC, which has a record length of 112.

```
**free

snd-msg 'The SND-MSG operation sends an informational message or an exception message.';

*inlr = *on;
return;
```

I don't save the source, I don't try to compile. I now simply close VS Code.

When I open VS Code again I see the tab for my new member with the following message.

![image](https://user-images.githubusercontent.com/7809440/231564252-5bfecb9f-8ce2-41dd-b0fb-cd6bba9d7c3e.png)

I don't click Try Again yet.

I reconnect to my IBM i server and then click Try Again.

I then compile, which does an auto save, and it fails.

![image](https://user-images.githubusercontent.com/7809440/231564287-c48dc497-80d4-40f5-b526-d611ce1dfce5.png)

If I look in PDM I can see the line has been truncated.

![image](https://user-images.githubusercontent.com/7809440/231564328-c2818a1a-f2a2-4817-9895-a71de3d9f5ab.png)

I'm not sure this is the only way to replicate the problem but I wonder if you don't open a member via the object browser then maybe the record length isn't being retrieved.

**Comment by worksofliam** [comment]
This is where it is happening: https://github.com/halcyon-tech/vscode-ibmi/blob/master/src/filesystems/qsys/extendedContent.ts#L121

I suspect because the document remains open unsaved between instances, it doesn't have to open the member again on the second instance, therefore we don't have the record length and we default to 80.

**Comment by richardm90** [comment]
I'm working on a fix for this, I think I'm pretty close to a solution.

**Comment by richardm90** [comment]
This issue has been fixed and addressed as part of pull request #1389.
